src/core_plugins/elasticsearch/index.js

```diff
@@ -35,6 +37,9 @@ module.exports = function ({ Plugin }) {
           key: string()
         }).default(),
         apiVersion: Joi.string().default('master'),
+        healthCheck: object({
+          delay: number().default(2500)
```

Comment:
Not set on adding, but needed to make it configurable for the tests to complete faster. I have heard of users manually changing the constant in `health_check.js`.